templates/docs/examples/patterns/navigation/search-dark.html

```diff
@@ -0,0 +1,57 @@
+{% extends "_layouts/examples.html" %}
+
+{% block title %}Navigation / Search dark{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
+
+{% block content %}
+
+<header id="navigation" class="p-navigation is-dark">
+  <div class="p-navigation__row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__logo">
+        <a class="p-navigation__item" href="#">
+          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/3c7954dd-logo-canonical-white.svg" alt="" width="95" />
+        </a>
+      </div>
+      <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav" aria-label="Example main navigation">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item is-selected">
+          <a class="p-navigation__link" href="#">Products</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">Services</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">Partners</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">About</a>
+        </li>
+      </ul>
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item u-hide--small" style="display: flex; align-items: center;" >
+          <a href="#" class="js-search-button p-navigation__link p-navigation__search--toggle is-light">
+            <span class="u-hide u-show--large" style="padding-right:1rem">Search</span>
+          </a>
+        </li>
+      </ul> 
+      <div class="p-navigation__search u-show--small">
```

Comment:
It seems there is some weird edgecase at 772px wide screens:

<img width="449" alt="Screenshot 2021-10-01 at 15 54 18" src="https://user-images.githubusercontent.com/83575/135632112-c7f5059f-774f-4241-a056-7feb616be1e4.png">

The search bar is already visible but navigation is not yet in mobile state.
There must be some 1px difference in how breakpoints are defined. Maybe one part uses min-width and the other max-width.